package.json

```diff
@@ -28,9 +28,6 @@
     "email": "ismaelga@gmail.com"
   },
   "main": "dist/react-json-editor.js",
-  "peerDependencies": {
-    "react": ">=0.12.2 <1.0.0"
-  },
```

Comment:
@newbreedofgeek Projects using react-json-editor will already have their own React and this module will need to support that version rather than use its own. Hence it's a peer dependency.